bazel/emscripten_toolchain/env.bat

```diff
@@ -3,3 +3,5 @@
 set ROOT_DIR=%CD%
 set EMSCRIPTEN=%ROOT_DIR%\%EM_BIN_PATH%\emscripten
 set EM_CONFIG=%ROOT_DIR%\%EM_CONFIG_PATH%
+
+set PYTHONSAFEPATH=
```

Comment:
Missing newline here and in the other file.

Reply:
Added in 8a9ba848c1b916ac648ba5584f50bd0b14c05f52